api/server/router/delta/delta_routes.go

```diff
@@ -0,0 +1,27 @@
+package delta
+
+import (
+	"net/http"
+
+	"github.com/docker/docker/api/server/httputils"
+	"github.com/docker/docker/api/types"
+	"golang.org/x/net/context"
+)
+
+func (d *deltaRouter) postDeltasCreate(ctx context.Context, w http.ResponseWriter, r *http.Request, vars map[string]string) error {
+	if err := httputils.ParseForm(r); err != nil {
+		return err
+	}
+
+	deltaSrc := r.Form.Get("src")
```

Comment:
I guess it's ok if `DeltaCreate` can work with empty strings to see [here](https://github.com/golang/go/blob/b3b9b5e463cefb5eaad69628709ed5a0c1cb658f/src/net/url/url.go#L772-L775)